DataPreprocessing.py

```diff
@@ -0,0 +1,39 @@
+"""
```

Comment:
The code here is too free form, will be good to wrap this into better objects or functions

Reply:
Addressed this issue in the new branch (see new pull req)